### ISSUE TITLE
zfdatagrid in many to many relations?

### ISSUE THREAD

**Issue by imonteiro** [zfdatagrid/grid#628]
_Original author: nowicki....@gmail.com (May 03, 2010 11:02:33)_

Hi,

I think that's the last newbie's question.

I wonder (and desperately looking for solution) if there is any wany to use
zfdatagrid for instance in case i attached as print screen.
Is it possible to let user add multiple parameters  to equipment type?

Next step would be to add this type to equipment.

Probably I would go round avoiding equipment _parameter and connect
parameter directly with equipment via another table.

That's the problem which does not let me sleep

I am very thankful for any suggestions.

Kind Regards,

Adam

_Original issue: http://code.google.com/p/zfdatagrid/issues/detail?id=272_

**Comment by imonteiro** [comment]
_From bento.vi...@gmail.com on May 14, 2010 14:52:11_
Hi,

Are you talking about adding multiple params at once?

Best Regards,
Bento Vilas Boas

**Comment by imonteiro** [comment]
_From nowicki....@gmail.com on May 17, 2010 20:27:44_
Hello,

Yes, exacly

I handled this by working with standard form and checkboxes however it took a lot of
time for me to prepare it first time. I noticed that (perhaps I should not say that
:))other frameworks sometimes implement it and generate such a solution automaticaly.
I was just wondering if there is a chance to do so with zfdatagrid which I love(i
need more time to understand all features etc.).

Thank you for your time and contribution.
